src/matrix/inversewishart.jl

```diff
@@ -43,6 +43,16 @@ size(d::InverseWishart) = (p = dim(d); (p, p))
 params(d::InverseWishart) = (d.df, d.Ψ, d.c0)
 @inline partype{T<:Real}(d::InverseWishart{T}) = T
 
+### Conversion
+function convert{T<:Real}(::Type{InverseWishart{T}}, d::InverseWishart)
+    P = convert_eltype(T, d.Ψ)
```

Comment:
One more question related to that: it's trivial to define the `convert` for a `ZeroVector`, but since `ZeroVector <: Any`, I can't just `convert(AbstractArray{T}, d.μ)`. Any ideas for a clean way to do that using dispatch while still getting rid of my little helper function?